### PR TITLE
[move] Removed scripts from transactional tests

### PR DESCRIPTION
**Pull request by tnowacki**
## Description 

- In prep for killing scripts, rewrote most of the tests using them (many more to come...)

## Test Plan 

- Ran tests

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: external-crates/move/crates/move-transactional-test-runner/src/framework.rs [hunk 1]
```diff
@@ -182,33 +173,37 @@ pub trait MoveTestAdapter<'a>: Sized + Send {
...
+                    let source_mapping = match source_map {
+                        Some(m) => SourceMapping::new(m, view),
+                        None => SourceMapping::new_from_view(view, Spanned::unsafe_no_loc(()).loc)
+                            .expect("Unable to build dummy source mapping"),
+                    };
+                    let disassembler =
+                        Disassembler::new(source_mapping, DisassemblerOptions::new());
+                    merge_output(output, Some(disassembler.disassemble().unwrap()))
+                });
+                Ok(output)
```
Comment (cgswords) [inline external-crates/move/crates/move-transactional-test-runner/src/framework.rs:206]: :tada:

File: external-crates/move/crates/move-transactional-test-runner/src/framework.rs [hunk 2]
```diff
@@ -493,6 +516,12 @@ impl<'a> CompiledState<'a> {
     }
 }
 
+pub struct MaybeNamedCompiledModule {
+    pub named_address: Option<Symbol>,
+    pub module: CompiledModule,
+    pub source_map: Option<SourceMap>,
+}
```
Comment (cgswords) [inline external-crates/move/crates/move-transactional-test-runner/src/framework.rs:523]: This change seems nicer long-term, but it did lead to a lot of code churn. 😢

File: external-crates/move/crates/move-transactional-test-runner/src/framework.rs [hunk 3]
```diff
@@ -240,42 +235,43 @@ pub trait MoveTestAdapter<'a>: Sized + Send {
...
+                    command_lines_stop,
+                    stop_line,
+                    data,
+                    |adapter, modules| {
+                        adapter.publish_modules(modules, gas_budget, empty_publish_args)
+                    },
+                )
+                .await?;
+                let (module_id, name) = single_entry_function(&modules).unwrap_or_else(|err| {
+                    panic!("{} on lines {}-{}", err, start_line, command_lines_stop)
```
Comment (cgswords) [inline external-crates/move/crates/move-transactional-test-runner/src/framework.rs:255]: Would it be possible to add a note to explain `run` expects on failure?

**Review by cgswords — APPROVED** [review]
LGTM. Thanks for doing this, and fixing the bytecode printer.

File: external-crates/move/crates/move-transactional-test-runner/src/framework.rs (hunk 3 above)
Comment (tnowacki) [reply external-crates/move/crates/move-transactional-test-runner/src/framework.rs:255]: The single_entry_function error gives you that. Let me know if you want anything more and I can add it in another PR :)

Comment (cgswords) [reply external-crates/move/crates/move-transactional-test-runner/src/framework.rs:255]: Er, I meant explicitly explaining that we "Expected exactly one function or one entry function" _because_ we're in **run**.

**Pull request merged**